### PR TITLE
Initialize repo

### PR DESCRIPTION
**Pull request by electrocnic**

### CODE REVIEW

File: src/Main.cpp [hunk 1]
```diff
@@ -88,6 +88,11 @@ int main(int argc, char** argv)
 	int window_height = 800;
 	bool fullscreen = false;
 	std::string window_title = "Task 0";
+	float field_of_view = 60.0f;
+	float near_plane_distance = 0.1f;
+	float far_plane_distance = 100.0f;
+	float aspect_ratio = window_width / window_height;
```
Comment (johannesugb) [inline src/Main.cpp]:
```suggestion
	constexpr float field_of_view = 60.0f;
	constexpr float near_plane_distance = 0.1f;
	constexpr float far_plane_distance = 100.0f;
	constexpr float aspect_ratio = window_width / window_height;
```

**Review by johannesugb — CHANGES_REQUESTED** [review]
What are the contents of the `/test` directory? 
These tests are not run automatically, are they? 
Are they of any use to users or just for our internal testing? If it is the latter, can we somehow hide them from users?

File: src/Main.cpp (hunk 1 above)
Comment (fknfilewalker) [reply src/Main.cpp]: why not constexpr?

**Comment by fknfilewalker** [comment]
> What are the contents of the `/test` directory? These tests are not run automatically, are they? Are they of any use to users or just for our internal testing? If it is the latter, can we somehow hide them from users?

They are not really tests. 
I am not really sure why we need them but if I remember correctly they are docker configs which we can use if someone on linux has problems with installing deps.

File: src/Main.cpp (hunk 1 above)
Comment (johannesugb) [reply src/Main.cpp]: Yeah, that's even better. Updated!

**Comment by johannesugb** [comment]
> I am not really sure why we need them but if I remember correctly they are docker configs which we can use if someone on linux has problems with installing deps.

Hmm so that means we should keep them?

**Comment by fknfilewalker** [comment]
> > I am not really sure why we need them but if I remember correctly they are docker configs which we can use if someone on linux has problems with installing deps.
> 
> Hmm so that means we should keep them?

I don't know, we could just copy the commands to install the deps in the readme, but in general people who are using linux normally know how to use linux so I wouldn't care too much about them, but let's see what @electrocnic thinks.

**Comment by johannesugb** [comment]
> > > I am not really sure why we need them but if I remember correctly they are docker configs which we can use if someone on linux has problems with installing deps.
> > 
> > 
> > Hmm so that means we should keep them?
> 
> I don't know, we could just copy the commands to install the deps in the readme, but in general people who are using linux normally know how to use linux so I wouldn't care too much about them, but let's see what @electrocnic thinks.

So.. @electrocnic what do you think?

**Comment by electrocnic** [comment]
I will incorporate those lines for each linux distro into the readme, perhaps?

And therefore also add hints for users which distros we officially support and which not.

**Pull request merged**